typescript-example/src/credentials.json

```diff
@@ -0,0 +1,6 @@
+{
+  "clientId": "",
```

Comment:
We can put these trial keys in the credentials for the user too https://developer.mappedin.com/guides/api-keys for mappedin-demo-mall, we usually do that for example apps

Reply:
Good call!